formats/tokens.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+const fs = require('fs');
+
+const colorFactory = require('./utils/color-factory/color-factory');
+const {tokensToJson} = require('./utils/color-factory/utils');
+
+function tokenify(scheme) {
```

Comment:
```suggestion
const tokenify = (scheme) => {
```

Reply:
All of us seem to prefer declarations (`function Thing() {}`) over expressions so let's use that consistently